### ISSUE TITLE
[🐛 BUG]: Service -  EPIPE (Broken pipe)

### ISSUE THREAD

**Issue by Grokon** [roadrunner-server/roadrunner#1130]
### No duplicates 🥲.

- [X] I have searched for a similar issue in our bug tracker and didn't find any solutions.

### What happened?

I create simple service and log output  is broken ( empty). Strace a php process is showing error: `-1 EPIPE (Broken pipe)`. It's happened  after upgrade rr from 2.9.1 to 2.9.2~2.10.0.rc3
rr.yaml:
```
rpc:
  listen: tcp://127.0.0.1:6001
service:
  qq:
    command: php qq.php
```
qq.php:
```
<?php
for ($x = 0; $x <= 1000; $x++) {
  sleep(1);
  error_log("The number is: $x", 0);
}
?>
```

### Version

roadrunner version >= 2.9.2
I'll try to run inside docker php:8.1.5-cli-alpine 
And WSL2 - Ubuntu 22.04 with PHP 8.1.2

### Relevant log output

```shell
~/tmp > ./rr-2.9.1 serve -d -c ./rr.yaml                                                                                                                                               
2022-05-12T08:31:44.467+0300    DEBUG   rpc             plugin was started      {"address": "tcp://127.0.0.1:6001", "list of the plugins with RPC methods:": ["service", "informer", "resetter"]}
[INFO] RoadRunner server started; version: 2.9.1, buildtime: 2022-04-11T10:26:46+0000
2022-05-12T08:31:44.473+0300    INFO    service         service have started    {"name": "qq", "command": "/usr/bin/php qq.php"}
2022-05-12T08:31:45.486+0300    INFO    service         The number is: 0
2022-05-12T08:31:46.486+0300    INFO    service         The number is: 1
2022-05-12T08:31:47.486+0300    INFO    service         The number is: 2
2022-05-12T08:31:48.486+0300    INFO    service         The number is: 3

~/tmp > strace -ewrite -p 4470
strace: Process 4470 attached
write(2, "The number is: 119\n", 19)    = 19
write(2, "The number is: 120\n", 19)    = 19
write(2, "The number is: 121\n", 19)    = 19

-------
~/tmp > ./rr-2.9.2 serve -d -c ./rr.yaml                                                                                                                                               
2022-05-12T08:34:01.916+0300    DEBUG   rpc             plugin was started      {"address": "tcp://127.0.0.1:6001", "list of the plugins with RPC methods:": ["resetter", "service", "informer"]}
[INFO] RoadRunner server started; version: 2.9.2, buildtime: 2022-04-28T23:42:35+0000
2022-05-12T08:34:01.918+0300    INFO    service         service have started    {"name": "qq", "command": "php qq.php"}
2022-05-12T08:34:02.933+0300    INFO    service         The number is: 0


~/tmp > trace -ewrite -p 5268
strace: Process 5268 attached
write(2, "The number is: 17\n", 18)     = -1 EPIPE (Broken pipe)
--- SIGPIPE {si_signo=SIGPIPE, si_code=SI_USER, si_pid=5268, si_uid=1000} ---
write(2, "The number is: 18\n", 18)     = -1 EPIPE (Broken pipe)
--- SIGPIPE {si_signo=SIGPIPE, si_code=SI_USER, si_pid=5268, si_uid=1000} ---
write(2, "The number is: 19\n", 18)     = -1 EPIPE (Broken pipe)
```

**Comment by rustatian** [comment]
@Grokon Hey 👋🏻. Thanks for the report. Could you please also include your configuration (`.rr.yaml`) and describe the way of creating services (via RPC or in the config)?

**Comment by rustatian** [comment]
@Grokon Nevermind, missed from the description 😆

**Comment by rustatian** [comment]
@Grokon ok, I was able to reproduce the problem. Fix will be in the `v2.10.0`.
